ETC_en.tsv

```diff
@@ -5503,13 +5503,13 @@ ETC_20150317_005502	Defeated 500 boss monsters.
 ETC_20150317_005503	Walking Dead
 ETC_20150317_005504	Achieved by resurrecting at least 100 times using NXP.
 ETC_20150317_005505	Crafting Beginner
-ETC_20150317_005506	Succeeded in crafting 10 times. Feels pretty good.
```

Comment:
Not sure if you thought this was a misspelling, but it works either way. The Korean strings say 'times' as in: crafted an item 10 times